src/main/java/com/aliyun/odps/jdbc/OdpsPreparedStatement.java

```diff
@@ -610,9 +617,23 @@ private String updateSql(String sql, HashMap<Integer, Object> parameters) throws
   }
 
   private String convertJavaTypeToSqlString(Object x) throws SQLException {
```

Comment:
Constants need some suffix. For example, TINYINTs should end with Y, decimals should end with BD.